README.md

```diff
@@ -85,6 +113,27 @@ yarn test:a11y
 
 Make sure all the paths in your application are covered by accessibility tests (see [a11y.ts](src/test/a11y/a11y.ts)).
 
+#### Cross browser tests
+
+Cross browser tests are run via [CodeceptJS](https://codecept.io/) using the [Playwright](https://playwright.dev/) plugin which tests Chrome/Edge, Safari and Firefox and [WebDriver](https://www.w3.org/TR/webdriver/) plugin which tests Internet Explorer via [Sauce Labs](https://saucelabs.com/).
+
+Cross browser tests are automatically run via the nightly Jenkins pipeline.
+
+Running cross browser tests locally, start the server then run:
+
+```bash
+# Playwright - Chrome/Edge, Safari and Firefox
+TEST_HEADLESS=false yarn test:crossbrowser:latest
+
+# WebDriver - Internet Explorer (via Sauce Labs)
+brew install sauce-connect
```

Comment:
linux pls

Reply:
👍 I've add a link to the install instructions in bfd8b64